crates/cmds-solana/src/nft/create_metadata_account.rs

```diff
@@ -189,7 +119,9 @@ pub enum Input {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
-pub struct InputStruct {
+pub struct Input {
+    #[serde(flatten)]
+    pub update_authority: UpdateAuthority,
```

Comment:
@enzotar you can use `#[serde(flatten)]` for this

Reply:
Ok